Vuzix/app/src/main/java/com/facilitation/view/activities/spotify/SpotifyListActivity.kt

```diff
@@ -0,0 +1,168 @@
+package com.facilitation.view.activities.spotify
+
+import BluetoothConnectionListener
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.bluetooth.BluetoothAdapter
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.KeyEvent
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.facilitation.view.R
+import com.facilitation.view.ViewApplication
+import com.facilitation.view.activities.MainActivity
+import com.facilitation.view.databinding.ActivitySpotifyListBinding
+import com.facilitation.view.model.TrackDTO
+import com.facilitation.view.receivers.TapReceiver
+import com.facilitation.view.utility.BluetoothHandler
+import com.facilitation.view.utility.ITapInput
+import com.facilitation.view.utility.MyActivityLifecycleCallbacks
+import com.facilitation.view.utility.SpotifyListAdapter
+import com.facilitation.view.utility.enums.TapToCommandEnum
+import com.google.gson.Gson
+import com.google.gson.reflect.TypeToken
+
+
+class SpotifyListActivity : AppCompatActivity(), BluetoothConnectionListener, ITapInput {
```

Comment:
BluetoothConnectionListener no longer needed because we initialise in ViewApplication